index.js

```diff
@@ -10,9 +10,39 @@ function parseGitUri(uri) {
     uri.match(/git@github.com:(.*)\/(.*)\.git/);
 }
 
+function verifyChangelog() {
+  var changelog = exec('git diff HEAD~1');
```

Comment:
I think this should be HEAD, since we haven't committed yet. Also, make sure you're only checking `CHANGELOG.md`.